antidox/pii_results.txt

```diff
@@ -1,16 +1,40 @@
-My phone number is (206) 555-0123.
+ Welcome to Wikipedia. Although everyone is welcome to make constructive contributions to Wikipedia, at least one of your recent edits, such as the one you made to Lahinch, did not appear to be constructive and has been '''automatically reverted''' by .  Please use the sandbox for any test edits you would like to make, and take a look at the welcome page to learn more about contributing to this encyclopedia.  '''If you believe there has been a mistake and would like to report a false positive, please  and then remove this warning from your talk page.'''  If your edit was not vandalism, please feel free to make your edit again after reporting it. The following is the log entry regarding this warning: Lahinch was changed by    making a minor change with obscenities on 2007-10-16T16:12:34+00:00 . Thank you.  
```

Comment:
Can this file and `toxicity_results.txt` be added to .gitignore and deleted from GitHub? If you get `git` errors trying to make that change in this PR it could be done in a later PR

Reply:
done